Wrong arguments to `om_target_tile`

### Describe the bug

It started with the following on g++ 13.3.0:
```
In member function ‘void std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::_M_dispose() [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’,
    inlined from ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::~basic_string() [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’ at /usr/include/c++/13/bits/basic_string.h:804:19,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_destroy() [with _Tp = std::__cxx11::basic_string<char>]’ at /usr/include/c++/13/optional:287:35,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = std::__cxx11::basic_string<char>]’ at /usr/include/c++/13/optional:318:14,
    inlined from ‘std::_Optional_payload<_Tp, false, _Copy, _Move>::~_Optional_payload() [with _Tp = std::__cxx11::basic_string<char>; bool _Copy = false; bool _Move = false]’ at /usr/include/c++/13/optional:441:65,
    inlined from ‘std::_Optional_base<std::__cxx11::basic_string<char>, false, false>::~_Optional_base()’ at /usr/include/c++/13/optional:512:12,
    inlined from ‘std::optional<std::__cxx11::basic_string<char> >::~optional()’ at /usr/include/c++/13/optional:707:11,
    inlined from ‘void basecamp::start_cut_logs(const mission_id&, float)’ at /mnt/z/alef/src/faction_camp.cpp:2539:45:
/usr/include/c++/13/bits/basic_string.h:283:21: error: ‘*(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*)((char*)&<unnamed> + offsetof(const std::optional<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >,std::optional<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >::<unnamed>.std::_Optional_base<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, false, false>::<unnamed>)).std::__cxx11::basic_string<char>::<anonymous>.std::__cxx11::basic_string<char>::<unnamed union>::_M_allocated_capacity’ may be used uninitialized [-Werror=maybe-uninitialized]
  283 |           _M_destroy(_M_allocated_capacity);
      |           ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
/mnt/z/alef/src/faction_camp.cpp: In member function ‘void basecamp::start_cut_logs(const mission_id&, float)’:
/mnt/z/alef/src/faction_camp.cpp:2539:45: note: ‘<anonymous>’ declared here
 2539 |     tripoint_abs_omt forest = om_target_tile( omt_pos, 1, 50, terrains_forest,
      |                               ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 2540 |                               _( "Select a forest (or road/trail) from %d to %d tiles away." ) );
      |                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

What is really happening is an implicit conversion convertion from the `const char*` from `_( ... )` to the 5th `bool` argument of `om_target_tile()`.

It seems #78490 removed an arguments but the compilation succeded anyway and the error was not spotted.

### Attach save file

N/A

### Steps to reproduce

Build with g++ 13.3.0 (Ubuntu 24.04)

### Expected behavior

Add the missing arugments to `om_target_tile()` before the 8th and last argument `message` at the call site begin of `basecamp::start_cut_logs()`

### Screenshots

_No response_

### Versions and configuration

39fc5b2

### Additional context

_No response_

--- a/src/faction_camp.cpp
+++ b/src/faction_camp.cpp
@@ -2536,7 +2536,8 @@ static const std::unordered_set<oter_type_str_id> terrains_field_swamp_forest = 
 
 void basecamp::start_cut_logs( const mission_id &miss_id, float exertion_level )
 {
-    tripoint_abs_omt forest = om_target_tile( omt_pos, 1, 50, terrains_forest,
+    tripoint_abs_omt forest = om_target_tile( omt_pos, 1, 50, terrains_forest, true,
+                              tripoint_abs_omt::invalid, false,
                               _( "Select a forest (or road/trail) from %d to %d tiles away." ) );
     if( !forest.is_invalid() ) {
         standard_npc sample_npc( "Temp" );
@@ -2784,12 +2785,12 @@ void basecamp::start_fortifications( const mission_id &miss_id, float exertion_l
               "swamps are valid fortification locations.  In addition to existing fortification "
               "constructions." ) );
     tripoint_abs_omt start = om_target_tile( omt_pos, 2, 90, terrains_field_swamp_forest, true, omt_pos,
-                             _( "Select a start point from %d to %d tiles away." ) );
+                             false, _( "Select a start point from %d to %d tiles away." ) );
     if( start.is_invalid() ) {
         return;
     }
     tripoint_abs_omt stop = om_target_tile( omt_pos, 2, 90, terrains_field_swamp_forest, true, start,
-                                            _( "Select an end point from %d to %d tiles away." ) );
+                                            false, _( "Select an end point from %d to %d tiles away." ) );
     if( stop.is_invalid() ) {
         return;
     }

